File: Active-Assets/Active-Assets/lara/Lara.java
// Lara.java - Sniper explosiva de Valhalla Rising

public class Lara {
    private int criticalDamage;
    private int basicCooldown;
    private int ultimateCooldown;

    public Lara() {
        this.criticalDamage = 100;
        this.basicCooldown = 3;
        this.ultimateCooldown = 15;
    }

    public void headshot() {
        System.out.println("Headshot! -30% resistência física e mágica no alvo.");
    }

    public void desertEagleBurst(boolean sameTarget) {
        if (sameTarget) {
            System.out.println("Desert Eagle causa Dano Verdadeiro no último disparo!");
        } else {
            System.out.println("Disparo múltiplo com roubo de vida.");
        }
    }

    public void dispersiveShot() {
        System.out.println("Tiro Dispersivo! Dano Verdadeiro no centro da explosão.");
    }

    public void kabum(int kills) {
        if (kills >= 4) {
            this.ultimateCooldown = 0;
        } else if (kills == 3) {
            this.ultimateCooldown = 1;
        } else if (kills == 2) {
            this.ultimateCooldown = 2;
        } else if (kills == 1) {
            this.ultimateCooldown = 3;
        } else {
            this.ultimateCooldown = 15;
        }
        System.out.println("KABUM! Cooldown da ultimate: " + this.ultimateCooldown + "s");
    }
}

File: Active-Assets/Active-Assets/lara/lara.py
# lara.py - Sniper explosiva de Valhalla Rising

class Lara:
    def __init__(self):
        self.critical_damage = 100
        self.basic_cooldown = 3
        self.ultimate_cooldown = 15

    def headshot(self):
        print("Headshot! -30% de resistência física e mágica no alvo.")

    def desert_eagle_burst(self, same_target):
        if same_target:
            print("Desert Eagle: Dano Verdadeiro no último disparo!")
        else:
            print("Disparo múltiplo com roubo de vida.")

    def dispersive_shot(self):
        print("Tiro Dispersivo! Dano Verdadeiro no centro da explosão.")

    def kabum(self, kills):
        if kills >= 4:
            self.ultimate_cooldown = 0
        elif kills == 3:
            self.ultimate_cooldown = 1
        elif kills == 2:
            self.ultimate_cooldown = 2
        elif kills == 1:
            self.ultimate_cooldown = 3
        else:
            self.ultimate_cooldown = 15
        print(f"KABUM! Cooldown da ultimate: {self.ultimate_cooldown}s")
